.github/workflows/preview-release.yml

```
@@ -0,0 +1,88 @@
+name: Preview Release
+permissions: {}
+
+on:
+  pull_request:
+    branches: [main]
+    types: [labeled]
+
+# Automatically cancel in-progress actions on the same branch
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event_name == 'pull_request_target' && github.head_ref || github.ref }}
```

Do we need to update the trigger to `pull_request` here too or not?


```suggestion
  group: ${{ github.workflow }}-${{ github.event_name == 'pull_request' && github.head_ref || github.ref }}
```